projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap.sh

```diff
@@ -8,12 +11,16 @@ touch $SCRIPT_LOG
 # restore stdout and stderr at bottom of script
 exec 3>&1 4>&2 >>$SCRIPT_LOG 2>&1
 
-KUBE_VIP_IMAGE=$1
-VIP=$2
+# Check number of variables, given worker nodes don't need kube-vip
+if [ $# -eq 2 ]
```

Comment:
I wonder if we should have two scripts instead of branching here at the end of this one

Reply:
Both cp and work nodes use the same commands for network configuration, so I prefer to maintain a single script. A better solution could be to put the kube-vip manifest in the cluster deification yaml file,  like eks-a for vsphere. We will think about this when we know how large we can go with the user-data size.